sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/HDFSBackedStateStoreProvider.scala

```diff
@@ -135,17 +135,15 @@ private[sql] class HDFSBackedStateStoreProvider extends StateStoreProvider with
 
     /** Commit all the updates that have been made to the store, and return the new version. */
     override def commit(): Long = {
-      verify(state == UPDATING, "Cannot commit after already committed or aborted")
-
       try {
+        verify(state == UPDATING, "Cannot commit after already committed or aborted")
         commitUpdates(newVersion, mapToUpdate, compressedStream)
         state = COMMITTED
         logInfo(s"Committed version $newVersion for $this to file $finalDeltaFile")
         newVersion
       } catch {
-        case NonFatal(e) =>
-          throw new IllegalStateException(
-            s"Error committing version $newVersion into $this", e)
```

Comment:
Shall we check what is logged for `this`? Does it include more information about state store metadata like operator ID, partition ID, storeName, etc? If then might be better to keep it.

Reply:
Yeah this is a good point, it does capture operator/partition/etc. Updated to log `this` now.